Makefile

```
@@ -1,19 +1,19 @@
-IMAGE_NAME=koor-tech/genesis
-CONTAINER_NAME=koor-genesis
+IMAGE_PREFIX=koor-tech/genesis
 
-all: build run
+build-base:
+	docker build -t koor-tech/genesis-base-builder:latest -f base.Dockerfile .
 
-build: Dockerfile
-	@echo "Building Koor platform ..."
-	docker build -t $(IMAGE_NAME) .
+build-api:
+	@make build-base
```

Is there a reason it is not like this?

```suggestion
build-api: build-base
```

---

nop, great suggestion